src/helper_gitlab_modify.sh

```diff
@@ -177,21 +177,40 @@ get_gitlab_repo_if_not_exists_locally_and_exists_in_gitlab() {
   fi
 }
 
-# Structure:gitlab_modify
-# TODO: move to helper.
+
+#######################################
+# Performs a git pull inside the GitLab repository if the GitLab repository exists locally.
+# Local variables:
+#  gitlab_repo_name
+#  pwd_before
+#  pwd_àfter
+# Globals:
+#  MIRROR_LOCATION
+# Arguments:
+#   Name of the GitLab repository.
+# Returns:
+#  0 if funciton was evaluated succesfull.
+#  111 if the current path has't returned to what it origally was. 
+#  12 if the GitLab repository was not found locally. 
+# Outputs:
+#  FOUND if the GitLab repository exist locally.
+# TODO(a-t-0): move to helper.
```

Comment:
This TODO can be removed as the function has already been moved into a `helper....sh` file. (But I forgot to remove the TODO.)